lib/screenplay/alerts/alert.ex

```diff
@@ -1,204 +1,105 @@
 defmodule Screenplay.Alerts.Alert do
-  @moduledoc """
-  Represents a single Outfront Takeover Alert.
-  """
-
-  alias Screenplay.Alerts.State
-  alias Screenplay.Util
-
-  @enforce_keys [
-    :id,
-    :message,
-    :stations,
-    :schedule,
-    :created_by,
-    :edited_by,
-    :cleared_at,
-    :cleared_by
-  ]
-  defstruct @enforce_keys
-
-  @type id :: String.t()
-
-  @type canned_message :: %{
-          type: :canned,
-          id: non_neg_integer()
-        }
-
-  @type custom_message :: %{
-          type: :custom,
-          text: String.t()
-        }
-
-  @type station :: String.t()
-
-  @type schedule :: %{
-          start: DateTime.t(),
-          end: DateTime.t() | nil
-        }
-
-  @type user :: String.t()
-
-  @type update_map :: %{
-          optional(:message) => canned_message() | custom_message(),
-          optional(:stations) => list(station),
-          optional(:schedule) => schedule()
+  @moduledoc false
+
+  alias Screenplay.Alerts.Parser
+  alias Screenplay.V3Api
+
+  @derive Jason.Encoder
```

Comment:
Right right—I see that in the current code, we explicitly convert the struct to a map before passing it off to `Jason` to convert to a JSON string.

But the `@derive Jason.Encoder` line in this module says "this struct can be directly converted to JSON using `Jason`'s provided functions". If we actually tried `Jason.encode(%Alert{...})`, however, it would fail when trying to use its default encoder functions.

If we're not planning to ever call `Jason.encode` directly on an Alert struct, I'd recommend deleting the `@derive Jason.Encoder` line.